### PR TITLE
Updated libraries and linters

### PR DESCRIPTION
**Pull request by justin808**
Linters conform to ShakaCode internal project standards.

### CODE REVIEW

**Comment by justin808** [comment]
CC: @dylangrafmyre @jbhatab @alexfedoseev

**Comment by justin808** [comment]
@robwise Please confirm that newly generated code passes these linters.

File: client/.eslintrc [hunk 1]
```diff
@@ -6,13 +6,46 @@ extends: eslint-config-airbnb
 plugins:
   - react
 
+globals:
+  __DEBUG_SERVER_ERRORS__: true
+  __SERVER_ERRORS__: true
+  google: true
+  amenitiesList: true
+  File: true
```
Comment (alexfedoseev) [inline client/.eslintrc]: Let's remove `google`, `amenitiesList` & `File` from here, it's project specific.

Comment (justin808) [inline client/.eslintrc]: Done. I should have caught that.

**Pull request merged**